docs/content_management/pages/react_app_block.md

```diff
@@ -0,0 +1,118 @@
+---
+description: Create a block that allows an editor to embed a preconfigured React application into a page.
```

Comment:
I am not sure if we want to use React application or just React component, but maybe it is just synonym